add optional flags for devnode (#1658)
OffchainLabs/arbitrum-docs: arbitrum-docs/partials/_local-devnet-flags.md

<section class='small-table'>
| Flag                                     | Description                                                                                                                                                                                                                                                                                     |
| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `--build` | rebuild docker images                                                                                                                                                          |

Reviewer: ```suggestion
| `--build` | Rebuilds Docker images                                                                                                                                                          |
```

> | `--dev`                             | build nitro and blockscout dockers from source instead of pulling them. Disables simple mode                                                                                                                                                                                                 |

Reviewer: ```suggestion
| `--dev`                             | Builds Nitro and Blockscout Dockers from source instead of pulling them. Disables simple mode                                                                                                                                                                                                 |
```

> | `--init`                      | remove all data, rebuild, deploy new rollup                                                                                                                                                                                                            |

Reviewer: ```suggestion
| `--init`                      | Removes all the data, rebuilds, and deploys a new rollup                                                                                                                                                                                                            |
```

> | `--pos`                          | l1 is a proof-of-stake chain (using prysm for consensus)                                                                                                                                                                             |

Reviewer: ```suggestion
| `--pos`                          | L1 is a proof-of-stake chain (using Prysm for consensus)                                                                                                                                                                             |
```

> | `--validate`                            | heavy computation, validating all blocks in WASM                                                                                                                                                                                                                            |

Reviewer: ```suggestion
| `--validate`                            | Validates all blocks in WASM, heavy computation                                                                                                                                                                                                                    |
```

> | `--l3node`            | deploys an L3 node on top of the L2                                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `--l3node`            | Deploys an L3 node on top of the L2                                                                                                                                                                                                                                                    |
```

> | `--l3-fee-token`   | L3 chain is set up to use custom fee token. Only valid if also --l3node is provided                                                                                                                          |

Reviewer: ```suggestion
| `--l3-fee-token`   | Sets up the L3 chain to use a custom fee token. Only valid if `--l3node` flag is provided                                                                                                                          |
```

> | `--l3-fee-token-decimals`    | Number of decimals to use for custom fee token. Only valid if also --l3-fee-token is provided                                                                                                                                                                                                                         |

Reviewer: ```suggestion
| `--l3-fee-token-decimals`    | Number of decimals to use for a custom fee token. Only valid if `--l3-fee-token` flag  is provided                                                                                                                                                                                                                         |
```

> | `--l3-token-bridge`            | Deploy L2-L3 token bridge. Only valid if also --l3node is provided                                                                                                                                                                                                                                       |

Reviewer: ```suggestion
| `--l3-token-bridge`            | Deploys an L2-L3 token bridge. Only valid if `--l3node` flag is provided                                                                                                                                                                                                                                       |
```

> | `--batchposters`                | batch posters [0-3]                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `--batchposters`                | Batch posters [0-3]                                                                                                                                                                                                                        |
```

> | `--redundantsequencers`             | redundant sequencers [0-3]                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `--redundantsequencers`             | Redundant sequencers [0-3]                                                                                                                                                                                                                        |
```

> | `--detach`                             | detach from nodes after running them                                                                                                                                                                           |

Reviewer: ```suggestion
| `--detach`                             | Detaches from nodes after running them                                                                                                                                                                           |
```

> | `--blockscout`                           | build or launch blockscout                                                                                                                                                                                                                         |

Reviewer: ```suggestion
| `--blockscout`                           | Builds or launch the Blockscout                                                                                                                                                                                                                         |
```

> | `--simple`           | run a simple configuration. one node as sequencer/batch-poster/staker (default unless using --dev) |

Reviewer: ```suggestion
| `--simple`           | Runs a simple configuration. One node as a sequencer/batch-poster/staker (default unless using `--dev`) |
```

> | `--tokenbridge`    | deploy L1-L2 token bridge.                                                                                                                                                                   |

Reviewer: ```suggestion
| `--tokenbridge`    | Deploy an L1-L2 token bridge                                                                                                                                                                |
```

> | `--no-tokenbridge`    | don't build or launch tokenbridge |

Reviewer: ```suggestion
| `--no-tokenbridge`    | Opt out of building or launching the token bridge |
```

> | `--no-run`    | does not launch nodes (useful with build or init) |

Reviewer: ```suggestion
| `--no-run`    | Does not launch nodes (useful with build or init) |
```

> | `--no-simple`    | run a full configuration with separate sequencer/batch-poster/validator/relayer |

Reviewer: ```suggestion
| `--no-simple`    | Runs a full configuration with separate sequencer/batch-poster/validator/relayer |
```
